templates/emails/user_new_pro_account_admin_notification.tt.html

```diff
@@ -20,16 +20,18 @@
 <a href="mailto:[% user.email %]?subject=[% mail_subject_new_pro_account %]&cc=producteurs@openfoodfacts.org&body=[% mail_body_new_pro_account %]">E-mail de relance</a>
 <br>
 <br>
-
 [% IF user.requested_org_id %]
 <a href="mailto:[% user.email %]?subject=[% mail_subject_new_pro_account_org_request_validated %]&cc=producteurs@openfoodfacts.org&body=[% mail_body_new_pro_account_org_request_validated %]">E-mail de rattachement à l'organisation</a>
 <br>
 <br>
 [% END %]
-
+<br>
+<a href="https://connect.openfoodfacts.org/web#cids=1&menu_id=111&action=139&model=res.partner&view_type=list">Go to Connect</a>
+<br>
 Here is the tabular data for the producer if they are not in the tracking spreadsheet yet.
 <br>
-<br>Tracking spreadsheet: https://docs.google.com/spreadsheets/d/1Smx74TevmNlPyEqrcCtvN9uJIdUh-mC1cznLCQaV0V8/edit#gid=0
+<a href="https://docs.google.com/spreadsheets/d/1Smx74TevmNlPyEqrcCtvN9uJIdUh-mC1cznLCQaV0V8/edit#gid=0">Tracking spreadsheet</a>
```

Comment:
@manon-corneille 
Now that you made this link public, I had to change the settings to put a "limited access" to those having the link. (otherwise anyone with the link was able to write…)

Just telling you to help you remember everything you put on github is public !